app/assets/javascripts/Components/submission_table.jsx

```diff
@@ -357,6 +358,16 @@ class SubmissionsActionBox extends React.Component {
         </button>
       );
     }
+    if (this.props.is_ta) {
```

Comment:
Shouldn't this be conditional on whether they have permission to collect submissions (not just if they are a grader or not)?

Reply:
The collect submissions button will be visible to all graders. I am checking whether they have permission or not in submission_policy.rb. If they are not allowed, when they click the button, it will raise an error and won't allow them to collect. If we want to display the button based on their permission I can modify according to that.

Reply:
Yes, please only display pages/button/links/etc. to users who have permission to use them